### DISCUSSION TITLE
Install Help Needed

### DISCUSSION THREAD

**Discussion by W1LDxC4RD** [community/community#146561] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I hope I'm putting this in the correct category. If not, I apologize.
I'm new to coding and am trying to install from this: https://github.com/kmille/deezer-downloader. I am stumped on Step 2, though. I have forked the repository, as well as just downloading the zip. I have Git Bash, Python, and a Python IDE installed. On step 2, it says "pip install --user deezer-downloader." I assume that means to input that into the IDE, but when I do, it comes back with invalid syntax. And yes, I have tried changing where it says user to the username of the repository. I'm fairly intelligent, but I just can't figure out what I'm doing wrong. Thanks in advance for any help.

**Comment by MohammedMMC** [comment]
where did you run the command ? i think you should run it in the project folder console

**Reply by W1LDxC4RD** [reply]
I've tried to run it in a Python IDE called Thonny. Also tried it in the command prompt on Windows, just to see if it was easier than I was making it out to be. I will try it in the project folder console, but not exactly sure what that is. Do you mean in the project file on my local drive? If so, what would be the console? I should have learned coding years ago when I first got into computers, I guess. Maybe I'd understand it better.

**Reply by MohammedMMC** [reply]
You need to open a terminal or command prompt in the folder where the project files are saved.
Right-click inside the folder (but not on any file) select "Open in Terminal" or "Git Bash" and then run the command

**Reply by W1LDxC4RD** [reply]
Ok. I opened Git Bash in the folder and ran the command.  It came back saying command "pip" is unknown. I think I'll just stop trying to figure this out for now. Maybe sometime when I have more time to mess with it I'll try again.  I don't want to keep bothering you. Thank you. I really appreciate the help you've given me.

**Reply by MohammedMMC** [reply]
that's mean you need to add the PATH to your PC environment
i think that when you install python they ask you if you want to add it automatically 
this url may help you: https://stackoverflow.com/questions/23708898/pip-is-not-recognized-as-an-internal-or-external-command
